packages/reaction-sample-data/private/data/Shops.json

```diff
@@ -320,6 +320,10 @@
       "format": "%s%v",
       "symbol": "$"
     },
+    "PHP": {
+      "format": "%s%v",
+      "symbol": "PHP "
```

Comment:
I tried that and it seemed to throw off alignment. That's why there is a space after symbol. But maybe I am missing something.